octoprint_mrbeam/static/js/app/view-models/convert.js

```diff
@@ -585,6 +585,23 @@ $(function () {
                 Object.keys(tmp).length
             );
             self.custom_materials(tmp);
+            $(document).on("mouseover", ".material_entry", function () {
+                var $this = $(this);
```

Comment:
I would suggest to rename `$this ` with a significant name, like for example if it is the material entry or whatever, or to otherwise use `$(this)` in the rest of the code

Reply:
thx for the feedback, I added a name